build/__phpVersions.sh

```diff
@@ -1,7 +1,7 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
 PHP_RUNTIME_BASE_TAG='20220914.1'
-PHP_FPM_RUNTIME_BASE_TAG='20221004.1 '
+PHP_FPM_RUNTIME_BASE_TAG='20220914.1'
```

Comment:
This got updated because of this line: https://github.com/microsoft/Oryx/pull/1612/files#diff-47c28d7a6c8135707f46b624b5913e35beea6dfbe7a8be2db7efefde606eba59R166